site/content/docs/user/rootless.md

```diff
@@ -67,5 +67,10 @@ To create a kind cluster with Rootless Podman, just run:
 $ KIND_EXPERIMENTAL_PROVIDER=podman kind create cluster
 ```
 
+On some distributions, you might need to use systemd-run to start kind into its own cgroup scope:
```

Comment:
What distribution?

Reply:
At least Archlinux. Might be related to cgroupsv2/unified hierarchy